core/scripts/functions/templates/oracle.toml

```diff
@@ -43,6 +43,13 @@ maxSecretsSizesList = [10_240, 20_480, 51_200, 102_400, 307_200, 512_000, 1_048_
   updateFrequencySec = 30
   updateTimeoutSec = 10
 
+  [pluginConfig.OnchainSubscriptions]
+  blockConfirmations = 1
+  routerAddress = "{{router_contract_address}}"
```

Comment:
We have multiple places in the jobspec where there are duplicate definitions of the router contract address.  How hard would it be to exclusively use the router address specified under `contractID` everywhere?
(This is out of scope for this PR, but may be a good backlog ticket as it may add misconfiguration risk)

Reply:
Agreed, this would be a good improvement.

Reply:
Add a ticket to tech debt epic please!

Reply:
FUN-921